llo/plugin.go

```diff
@@ -89,9 +89,13 @@ type PredecessorRetirementReportCache interface {
 }
 
 const (
-	ReportFormatEVM  commontypes.LLOReportFormat = "evm"
-	ReportFormatJSON commontypes.LLOReportFormat = "json"
-	// Solana, CosmWasm, kalechain, etc... all go here
+	// NOTE: ReportFormat strings should be constrained to a maximum of 8 chars
```

Comment:
there is separate tooling that will set the config on-chain right? So we don't need to validate the length in mercury plugin?

Reply:
Yes